app/assets/javascripts/modules/track-links.js

```diff
@@ -5,7 +5,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
   'use strict'
 
   Modules.TrackLinks = function () {
-    this.start = function (element) {
+    this.init = function (element) {
       var category = element[0].getAttribute('data-track-category')
       var action = element[0].getAttribute('data-track-action')
       var links = element[0].getElementsByTagName('a')
```

Comment:
I think you can change `element[0]` to `element` in here as `init` takes a vanilla JS object and not a JQuery one - in GOV.UK JS-land. Worth checking with a FE dev.

Reply:
Yeah, this is an odd one - see my other comment about line 42 of the spec file. Ah yes, but that's probably the reason - this file was converted to vanilla JS, but the test was also updated, and confusion. Yes, try what @gclssvglx is suggesting, but modify the test to pass `element` instead of `[element]`.